docs/languages/java.md

```diff
@@ -151,7 +151,16 @@ One of the key advantages of VS Code is speed. When you open your Java source fi
   <source src="/docs/languages/java/intellisense.mp4" type="video/mp4">
 </video>
 
-For more details, see [Java Code Navigation and Editing](/docs/java/java-editing.md). VS Code also supports a range of [Refactoring](/docs/java/java-refactoring.md) and [Linting](/docs/java/java-linting.md) features.
+## Code snippets
+
+Visual Studio Code supports a wide range of popular Java code snippets to make you more productive, such as class/interface, syserr, sysout, if/else, try/catch, static main method. Using information from Java language server, it also provides a preview of the code snippet during the selection.
+
+For example, typing "**sout**" or "**sysout**" will directly produce a code snippet for `System.out.println()`; 
+Similarly, typing "**main**" or "**psvm**" will generate a code snippet for `public static void main(String[] args) {}`;
+
+We support a wide range of code snippet shortcuts and postfix completion features. To see the complete list, see [Java Code Navigation and Editing](/docs/java/java-editing.md). VS Code also supports a range of [Refactoring](/docs/java/java-refactoring.md) and [Linting](/docs/java/java-linting.md) features.
```

Comment:
`see [Java Code Navigation and Editing](/docs/java/java-editing.md).`

Can we directly link to the code snippet related part? sth like `/docs/java/java-editing.md#code-snippets`